.github/workflows/playwright.yml

```diff
@@ -29,8 +23,8 @@ jobs:
               run: npm ci
             - name: Create env file
               run: |
-                  touch .env
-                  echo DATABASE_URL=${{ vars.DATABASE_URL }} >> .env
+                  touch .env.local
+                  echo DATABASE_URL=${{ vars.DATABASE_URL }} >> .env.local
```

Comment:
@lamppu, instead of creating env-files, it would probably be a little clearer to just define the environment as is done [here](https://docs.github.com/en/actions/learn-github-actions/variables#defining-environment-variables-for-a-single-workflow).